### PR TITLE
Add random granuleID to IngestGranuleFtpSuccessSpec

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:**

Updates IngestGranuleFtpSuccessSpec to randomize the incoming granuleId.  The intermittent test failure reported in the test suggests that a situation exists where theGranuleResponse is returning without a bucket/key for all files. 

Added logging output to assist with debugging in the future.

Addresses [CUMULUS-1335](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1335)

### CODE REVIEW

File: example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js [hunk 1]
```diff
@@ -41,6 +43,10 @@ describe('The FTP Ingest Granules workflow', () => {
     console.log('\nStarting ingest test');
     inputPayload = JSON.parse(fs.readFileSync(inputPayloadFilename, 'utf8'));
     inputPayload.granules[0].dataType += testSuffix;
+    inputPayload.granules[0].granuleId = randomStringFromRegex(granuleRegex);
```
Comment (ifestus) [inline example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js]: idk if it makes sense to use this here, but there's a `setupTestGranuleForIngest` function in `helpers/granuleUtils` that may be worth looking at...?

Comment (ifestus) [reply example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js]: https://github.com/nasa/cumulus/blob/master/example/spec/helpers/granuleUtils.js#L47

Comment (ifestus) [reply example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js]: nvm - looking further into it this really wouldn't offer any benefit to what you're currently doing. Disregard 😄

Comment (Jkovarik) [reply example/spec/parallel/ingestGranule/IngestGranuleFtpSuccessSpec.js]: Yeah, I looked at that, but the random granule function was the *only* thing it was doing we cared about.

**Pull request merged**